locales-pending/landing-premium.ftl

```diff
@@ -101,3 +101,12 @@ landing-premium-plans-cards-feature-removal-plus = <b>Automatic data removal</b>
 landing-premium-plans-cards-feature-alerts = Get alerts when your data has been breached
 landing-premium-plans-cards-feature-guidance = <b>Guided help</b> to fix high-risk data breaches
 landing-premium-plans-cards-feature-monitoring = Continuous monitoring
+
+# US FAQ
+
+landing-premium-what-websites-sell-info-qn = What kinds of websites sell my personal information?
+landing-premium-what-websites-sell-info-ans = Certain websites are in the business of collecting and selling people’s personal information without their consent, which is unfortunately legal in the US. These sites are called data brokers and they make up a $240 billion dollar industry. They use sophisticated methods to collect personal, financial, location, and even health information, often without your consent or even your knowledge. They’ll sell what they’ve collected to third parties, profiting from your information and leaving you open to violations of your privacy and security.
+landing-premium-continuous-data-removal-qn = What kinds of websites sell my personal information?
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+landing-premium-continuous-data-removal-ans = Every month, we use the information you provided about yourself (name, location and birthdate) to search across { $data_broker_sites_total_num } data broker sites that sell people’s private information. If we find your data on any of these sites, we initiate the request for removal. Data removal can take anywhere from a day to a month. This feature is available for { -brand-monitor-plus } users only. <learn_more_link>Learn more here.</learn_more_link>
```

Comment:
I'm not sure `-brand-monitor-plus` is a valid term (looking at the CI failed tests). The <kbd>grep</kbd> only returns these two new results, but no definitions.

```sh
git grep "\-brand-monitor-plus" | cat

locales-pending/landing-premium.ftl:landing-premium-continuous-data-removal-ans = Every month, we use the information you provided about yourself (name, location and birthdate) to search across { $data_broker_sites_total_num } data broker sites that sell people’s private information. If we find your data on any of these sites, we initiate the request for removal. Data removal can take anywhere from a day to a month. This feature is available for { -brand-monitor-plus } users only. <learn_more_link>Learn more here.</learn_more_link>
locales-pending/premium.ftl:premium-upsell-dialog-title = Turn on automatic data removal with { -brand-monitor-plus }
```